include/ply/syscall.h

```diff
@@ -34,4 +34,15 @@ int bpf_map_next  (int fd, void *key, void *next_key);
 int perf_event_open(struct perf_event_attr *hw_event, pid_t pid,
 		    int cpu, int group_fd, unsigned long flags);
 
+#ifndef offsetof
+#define offsetof(TYPE, MEMBER) ((size_t) &((TYPE *)0)->MEMBER)
+#endif
+
+#ifndef offsetofend
+#define offsetofend(TYPE, FIELD) \
+	(offsetof(TYPE, FIELD) + sizeof(((TYPE *)0)->FIELD))
+#endif
```

Comment:
We shouldn't need this. Please remove.

See comment on `bpf_prog_test_run`

Reply:
OK, Fixed in [93bbc1b](https://github.com/iovisor/ply/pull/82/commits/93bbc1b022b4c2587e0b32aa3fd52bb752da5e2a) .